v8.6.0 changelog and README updated (#597)
dell/dellemc-openmanage-ansible-modules: CHANGELOG.rst

.. contents:: Topics

v8.6.0
======

Release Summary
---------------

- Support for environment variables as fallback for credentials for all modules.

> - Enhanced idrac_certificates to support `CUSTOMCERTIFICATE` and upload of SSL key.

Reviewer: ```suggestion
- Enhanced idrac_certificates module and idrac_certificate role to support `CUSTOMCERTIFICATE` and `HTTPS` certificate import with SSL key.
```

Major Changes
-------------

- All OME modules are enhanced to support environment variables `OME_USERNAME` and `OME_PASSWORD` as fallback for credentials.
- All iDRAC and redfish modules are enhanced to support environment variables `IDRAC_USERNAME` and `IDRAC_PASSWORD` as fallback for credentials.

> - idrac_certificates - The module is enhanced to support import and export of `CUSTOMCERTIFICATE`.

Reviewer: ```suggestion
- idrac_certificates module and idrac_certificate role - This is enhanced to support import and export of `CUSTOMCERTIFICATE`.
```

Minor Changes
-------------

> - idrac_certificates - The `email_address` is no longer a required parameter.

Reviewer: ```suggestion
- idrac_certificates module and idrac_certificate role - The `email_address` has been made an optional parameter.
```

> - idrac_certificates - The module and role are enhanced to support upload of SSL key along with `HTTPS` certificate.

Reviewer: ```suggestion
- idrac_certificates - The module and role are enhanced to support import of `HTTPS` certificate with SSL key.
```

Bugfixes
--------

- Fixed an issue where the `NO_PROXY` environment variable was being ignored. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/554)

> - idrac_certificates - The `email_address` is no longer a required parameter. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/582).

Reviewer: ```suggestion
- idrac_certificates - The `email_address` has been made an optional parameter. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/582).
```
